src/Forms/Shared/Samples/Layers/OpenStreetMapLayer/OpenStreetMapLayer.xaml.cs

```diff
@@ -29,7 +29,7 @@ public OpenStreetMapLayer()
         private void Initialize()
         {
             // Create the OpenStreetMap basemap.
-            Basemap osmBasemap = Basemap.CreateOpenStreetMap();
```

Comment:
Previously this sample was showing content from the OpenStreetMap service directly, now we're showing an ArcGIS service built with OSM data; we might want to think about what specifically this sample is meant to show. I suspect readme updates may be in order here as well.